services/app-web/src/pages/StateSubmission/RateDetails/SingleRateCert/SingleRateCert.tsx

```diff
@@ -171,18 +171,17 @@ export const SingleRateCert = ({
                             >
                                 Document definitions and requirements
                             </Link>
-                            <span className="padding-top-1">
-                                Upload one rate certification only.
-                            </span>
-                            <span>
-                                {supportingDocsByRate
-                                    ? 'Additional rates can be added later.'
-                                    : 'Additional rates and supporting documents can be added later.'}
+                            <span className="padding-top-2">
+                                {`Upload only one rate certification document. ${
+                                    supportingDocsByRate
+                                        ? 'Additional rates can be added later.'
+                                        : 'Additional rates and supporting documents can be added later.'
+                                }`}
                             </span>
 
                             <span className="padding-top-1">
-                                This input only accepts PDF, CSV, DOC, DOCX,
-                                XLS, XLSX, XLSM files.
+                                This input only accepts one file in PDF, DOC, or
+                                DOCX format.
                             </span>
                         </span>
                     }
```

Comment:
We also need to change the constant passed into `accept` prop of the `FileUpload`. This is currently set to allow PDF, CSV, DOC, DOCX, XLS, XLSX, XLSM files. We should make a new constant in the same `constants.ts` file that would be for rate certification documents.

The constants file is in `services/app-web/src/components/FileUpload/constants.ts`

Reply:
Thanks for taking a look Jason, I updated the PR accordingly